# 17-06-16 at 14:28  -  Exceptions catch in ~Room()
